tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
these all seem like `let`.

`const` should be reserved for things that absolutely must stay constant, not `let` that  happen to remain constant.
Individuals should feel compelled to NOT change `const` -> `let`, over utilizing it degrades the value of this messaging.